VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/models/linked-participant-panel-model.spec.ts

```diff
@@ -178,32 +324,3 @@ describe('LinkedParticipantPanelModel', () => {
         model = LinkedParticipantPanelModel.forJudicialHolders(pats, roomLabel, roomId);
```

Comment:
same with this function, it appeared to be used before, but is it being used anywhere other than those tests?